src/pybind/ceph_argparse.py

```diff
@@ -327,7 +327,7 @@ class CephString(CephArgtype):
     """
     String; pretty generic.  goodchars is a RE char class of valid chars
     """
-    def __init__(self, goodchars=''):
+    def __init__(self, goodchars='', allowempty=True):
```

Comment:
We want to allow empty strings by default? Maybe we can keep this false by default?

Reply:
The default behavior is to allow empty strings.

Reply:
I intend to keep the default behavior and not cause the tests grind to a halt due to changes to the default behavior.